scripts/create_newcase

```diff
@@ -168,11 +164,19 @@ OR
     if args.input_dir is not None:
         args.input_dir = os.path.abspath(args.input_dir)
 
+    if args.ninst_couplers:
+        ncouplers = args.ninst
+        ninst = 1
```

Comment:
Shouldn't this set `ninst = args.ninst`?

I tried `./create_newcase --ninst 3 --ninst-couplers --case test_ninst_0621b --compset A --res f45_g37_rx1`

and got:

```
[roo2:~/cime/scripts/test_ninst_0621b]$ ./xmlquery NINST
	NINST: ['CPL:3', 'ATM:1', 'LND:1', 'ICE:1', 'OCN:1', 'ROF:1', 'GLC:1', 'WAV:1', 'ESP:1']
```

whereas I expected NINST to be 3 for everything except for ESP.